packages/graphql/src/classes/subscription/Neo4jGraphQLSubscriptionsCDCEngine.ts

```diff
@@ -97,7 +103,16 @@ export class Neo4jGraphQLSubscriptionsCDCEngine implements Neo4jGraphQLSubscript
     }
 
     private async pollEvents(): Promise<void> {
-        const cdcEvents = await this.cdcApi.queryEvents(this.subscribeToLabels);
+        let txFilter: Cypher.Map | undefined;
+        if (this.filterGraphQLEvents) {
+            const app = `${environment.NPM_PACKAGE_NAME}@${environment.NPM_PACKAGE_VERSION}`;
```

Comment:
This is a value used elsewhere, so we should have it as a constant somewhere to ensure these don't fall out of sync.